### PR TITLE
Add BreadCrumb with year selector

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/JHKHOrzX/294-feature-budget-summary-navigation

# What solved
- [X]: Should create a new BreadCrumb navigation with year-selector

# Description
 Should create a new BreadCrumb navigation with year-selector

### CODE REVIEW

File: src/stories/components/svg/select-chevron-down.tsx [hunk 1]
```diff
@@ -5,14 +5,26 @@ interface SelectChevronDownProps {
...
+  ...props
+}: SelectChevronDownProps) => (
   <svg
+    className={className}
+    onClick={onClick}
     width={`${width}px`}
     height={`${height}px`}
     viewBox="0 0 10 6"
     fill={fill}
+    cursor={onClick ? 'pointer' : 'none'}
```
Comment (YasielCabrera) [inline src/stories/components/svg/select-chevron-down.tsx:27]: `cursor: none` means hide the cursor, should be `default` the expected behavior right?

File: src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import BreadcrumbYearNavigation from './BreadcrumbYearNavigation';
+import type { NavigationBreadCrumb } from '@ses/components/Breadcrumbs/Breadcrumbs';
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx:3]: The correct naming would be `NavigationBreadcrumb` (without the capital C)

File: src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+const args = [
+  {
+    years: ['2022', '2023'],
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    handleChange: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onOpen: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onClose: () => {},
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx:27]: use `() => null` instead, this way you don't need to disable eslint checks

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/svg/select-chevron-down.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/svg/select-chevron-down.tsx]: you are right, done

File: src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx (hunk 3 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx]: done

File: src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx]: For this , you will see more files change, but just correct the name in all of them
Done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**